crates/sui-mvr-graphql-rpc/src/test_infra/cluster.rs

```diff
@@ -19,7 +19,7 @@ pub use sui_indexer::config::SnapshotLagConfig;
 use sui_indexer::errors::IndexerError;
 use sui_indexer::store::PgIndexerStore;
 use sui_indexer::test_utils::start_indexer_writer_for_testing_with_mvr_mode;
-use sui_pg_temp_db::{get_available_port, TempDb};
+use sui_pg_db::temp::{get_available_port, TempDb};
```

Comment:
digging the rename, but do we foresee other modules beyond temp?

Reply:
I ended up putting the `Db` type into this crate, so that's why the name changed. So what was the main module in `sui_pg_temp_db` has become the `sui_pg_db::temp` and the `Db` type has moved into the root module of `sui_pg_db`.